scripts/benchmark_node_analyze_readme.txt

```diff
@@ -0,0 +1,23 @@
+Steps to do the node scanning benchmark
+
+Pre steps:
+
+    1.Make sure docker is installed on your local laptop
+    2.execute docker run -it --rm registry.access.redhat.com/ubi8/ubi
+    3.make sure Vim, RPM and RPM-build are installed in the running ubi8 bash
+        if not, do yum search <package name> and yum install <package name>
```

Comment:
Does it worth creating a `Dockerfile` to generate this test image and upload it to the public-facing quay repo? I am thinking of a test image. We wouldn't need to execute this setup to run the tests; one step towards automating it.

Reply:
Yeah, I agree with the idea of creating a Dockerfile in quay repo.

Reply:
not 100% sure we have public-facing images. Perhaps we can start this? I believe there is a `quay.io/stackrox-io/qa` repo, but I don't think it's populated

Reply:
Having a minimal test image with all prereqs would be a great idea!
That could also make it easier to automate. In terms of registries, let's start this.

@RTann are you maybe thinking of `https://quay.io/repository/rhacs-eng/qa`? That one isn't public, though.
But we could set up a `qa` or `testing-infrastructure` repo that could hold this kind of images.